qutebrowser/app.py

```diff
@@ -723,6 +726,7 @@ def _shutdown(self, status, restart):
         # Now we can hopefully quit without segfaults
         log.destroy.debug("Deferring QApplication::exit...")
         objreg.get('signal-handler').deactivate()
+        objreg.get('session-manager').session_delete('_autosave', force=True)
```

Comment:
`session_delete` is the command handler - this should call `.delete('_autosave')` instead (which doesn't need `force=True`), and make sure to handle `SessionError` and `OSError` (probably by just logging the exception).